src/ebay-button/button-expand.tsx

```diff
@@ -1,15 +1,20 @@
-import React, { FC } from 'react'
+import React, { FC, ReactNode } from 'react'
 import EbayButtonCell from './button-cell'
 import EbayButtonText from './button-text'
 import { EbayIcon } from '../ebay-icon'
 
-const EbayButtonExpand: FC = ({ children }) =>
+type Props = {
+    children: ReactNode;
+    showDropdown?: boolean;
+}
+
+const EbayButtonExpand: FC<Props> = ({ children, showDropdown }) =>
     children ? (
         <EbayButtonCell>
             <EbayButtonText>
                 {children}
             </EbayButtonText>
-            <EbayIcon name="dropdown" />
+            {showDropdown && <EbayIcon name="dropdown" />}
```

Comment:
@abiyasa Can you explain to me how we hide the dropdown Icon here?

Reply:
@don-siu Wait, an expand button without its expander icon is just a button, no?

I think something is not right or not necessary here. You modified ebay-button so you can display the expander button without its expander icon, only the overflow icon, right? If so, you dont have to do that anymore.

If you use the latest ebay-fake-menu-button variants overflow, it already does that for you (or check the [latest ebay-ui-core-react storybook](https://opensource.ebay.com/ebayui-core-react/main/index.html?path=/story/ebay-fake-menu-button--variants)):

![Screenshot 2022-12-01 at 09 43 15](https://user-images.githubusercontent.com/1072047/205007664-6c92f143-e919-4750-979f-c616a6cc1bbd.png)

I would suggest you to remove completely `showDropdown`:

- pull the latest main branch
- Undo your changes on ebay-button
- Remove showDropdown from EbayFakeMenuButton

Reply:
updated and fixed